pcb/README.md

```diff
@@ -17,6 +14,14 @@ submission; no assembly instructions are provided here.
 The RespiraWorks Ventilator Mainboard PCB is a custom component that integrates the Cycle Controller, UI Computer,
 sensors, actuator drives, power supplies, filtering, and protection components.
 
+Note the the information below currently pertains to the Rev 1 PCB, done in Alitum.  For the next rev, Rev A, the project is currently moving to KiCad for better accessibility.  This Readme will transition over to supporting the new KiCad-based design as it develops.
+
+The exported complete Rev 1 files for manufacture can be found in the folder [rev1_export](rev1_export).
+
+The old Altium-based design files, including a snapshot of this readme, can be found in the folder [archived-Altium-Version-RevA-work-in-progress](archived-Altium-Version-RevA-work-in-progress)
+
+
+
 ## Contents
 
```

Comment:
Some content below might be irrelevant and/or redundant, particularly the identification of Altium files and the Rev 1.0 section

Reply:
I still want this to be visible so that anyone coming here to get info on the current PCB (i.e. someone debugging a ventilator) can find it)

Reply:
we will need to  update the readme as we move in to the KiCad project.  For now I just want to get the folder cleared so we can start the KiCad project